src/zjs_buffer.c

```diff
@@ -89,7 +92,9 @@ static jerry_value_t zjs_buffer_write_bytes(const jerry_value_t this,
         return zjs_error("zjs_buffer_write_bytes: invalid argument");
     }
 
-    uint32_t value = jerry_get_number_value(argv[0]);
+    // Cast a jerry_value_t to an uint32_t.
+    // info: jerry-core/ecma/base/ecma-helpers-value.c
+    uint32_t value = (uint32_t) (((int32_t) argv[0]) >> ECMA_DIRECT_SHIFT);
```

Comment:
Sorry, should have kept reviewing instead of jumping out to write my own sample code. :)